src/phoenix/core/model.py

```diff
@@ -149,15 +149,35 @@ def _get_embedding_dimensions(
     primary_embedding_features: Optional[
         EmbeddingFeatures
     ] = primary_dataset.schema.embedding_feature_column_names
-
     if primary_embedding_features is not None:
         embedding_features.update(primary_embedding_features)
+    primary_prompt_column_names: Optional[
+        EmbeddingFeatures
+    ] = primary_dataset.schema.prompt_column_names
+    if primary_prompt_column_names is not None:
+        embedding_features.update({"prompt": primary_prompt_column_names})
+    primary_response_column_names: Optional[
+        EmbeddingFeatures
+    ] = primary_dataset.schema.response_column_names
+    if primary_response_column_names is not None:
+        embedding_features.update({"response": primary_response_column_names})
+
     if reference_dataset is not None:
         reference_embedding_features: Optional[
-            Dict[str, EmbeddingColumnNames]
+            EmbeddingFeatures
         ] = reference_dataset.schema.embedding_feature_column_names
         if reference_embedding_features is not None:
             embedding_features.update(reference_embedding_features)
+        reference_prompt_column_names: Optional[
+            EmbeddingFeatures
+        ] = reference_dataset.schema.prompt_column_names
+        if reference_prompt_column_names is not None:
+            embedding_features.update({"prompt": reference_prompt_column_names})
+        reference_response_column_names: Optional[
+            EmbeddingFeatures
+        ] = reference_dataset.schema.response_column_names
+        if reference_response_column_names is not None:
+            embedding_features.update({"response": reference_response_column_names})
```

Comment:
`_get_embedding_dimensions` will recognize the prompt/response pair as embedding features

Reply:
nit: This won't matter much, but maybe we can add a check to let the user know that this being clobber: i.e. if they've added it to the new `prompt=/response=` pair as well as the original embeddings `dict` (since it's a `dict`, the check is pretty easy).

Reply:
I believe we should reserve the names `prompt` & `response` while they are considered embeddings features and part of the dictionary. What do you think?